python/az/aro/azext_aro/_validators.py

```diff
@@ -156,11 +156,14 @@ def _validate_subnet(cmd, namespace):
         if parts['child_type_1'].lower() != 'subnets':
             raise InvalidArgumentValueError(f"--{key.replace('_', '-')} child type '{subnet}' must equal subnets.")
 
-        client = get_mgmt_service_client(
-            cmd.cli_ctx, ResourceType.MGMT_NETWORK)
+        from .aaz.latest.network.vnet.subnet import Show
```

Comment:
If we put this here, every time we call this function, we have the overhead of sourcing this library function. Best to have all `import` statements at the top of the file so we only see that cost once.

Reply:
I agree 100%, the reason the import is not at the top is this is how upstream had it.  I'd be happy to change it to align more with what I did in _dynamic_validators.py for the imports, I wasn't sure how much we wanted to deviate from the upstream implementation. We could fix it in upstream since this is a bad practice to do with python.

Reply:
We can upstream whatever we'd like to our extension as long as they accept the PR, so I'd say if we're all on board here, let's try it

Reply:
Done, all imports are now at the top.